### PR TITLE
YML import and new schema naming refactoring (hot fix)

### PR DESCRIPTION
**Pull request by fboucquez**
New schemas, parser and exported yml update:
- fix: loading schemas from exported symbol.yml
- fix: updated java and typescript generators to new schemas names
- fix: update transactions helpers to use versions and types
- fix: updated vector tests to new network types.
- Disabled py and c++ generator from travis. There are maintained elsewere

Note, this PR doesn't go to the dev branch because the dev branch has typescript features for the new TS SDK, not for the old TS SDK. Examples are bigint support and constants.

Next PR will include mosaic revoke support (just the schema change and some vector tests)

### CODE REVIEW

**Comment by Jaguar0625** [comment]
can you please split this into (at least) two reviews and separate out the hotfix into its own review?

**Comment by fboucquez** [comment]
I'll split, but we would need both PRs, new schemas/yml import and the new revoke schemas + vector tests. I cannot just include the revoke schemas as they were coded into the new catbuffer schemas that use the new parsers with export/import.

**Comment by fboucquez** [comment]
@Jaguar0625 , this pr just includes the schema and parser upgrades for java and typescript. Revoke schemas will be on the next pr

File: generators/java/JavaHelper.py [hunk 1]
```diff
@@ -28,8 +28,10 @@ def get_load_from_binary_factory(self, attribute_class_name):
         return attribute_class_name
 
     def get_condition_operation_text(self, op):
-        if op == 'has':
+        if op == 'in':
             return '{0}.contains({1})'
+        if op == 'not equals':
+            return '{0} != {1}'
         return '{0} == {1}'
```
Comment (Jaguar0625) [inline generators/java/JavaHelper.py:35]: aren't you missing `not in` ?

File: generators/java/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+distributionUrl=https://services.gradle.org/distributions/gradle-5.6.4-bin.zip
```
Comment (Jaguar0625) [inline generators/java/gradle-wrapper.properties:1]: why is this file needed now?

File: test/vector/transactions.yml [hunk 3]
```diff
@@ -1,277 +1,277 @@
 - builder: MosaicMetadataTransactionBuilder
-  payload: B20000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000F6503F78FBF99544B906872DDB392F4BE707180D285E7919DBACEF2E9573B1E60000000001904442000000000000000001000000000000009083025FF3A8AB5AD104631FB370F290004952CD1FDDC4C90A00000000000000E8030000000000000A000600313233414243
+  payload: B20000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000F6503F78FBF99544B906872DDB392F4BE707180D285E7919DBACEF2E9573B1E60000000001984442000000000000000001000000000000009083025FF3A8AB5AD104631FB370F290004952CD1FDDC4C90A00000000000000E8030000000000000A000600313233414243
```
Comment (Jaguar0625) [inline test/vector/transactions.yml:2]: what is reason for changes in this file ?

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]

File: generators/java/gradle-wrapper.properties (hunk 2 above)
Comment (fboucquez) [reply generators/java/gradle-wrapper.properties:1]: Removed, for some reason, intellij keeps trying to download and use gradle 7. We haven't upgraded this generator yet.

File: generators/java/JavaHelper.py (hunk 1 above)
Comment (fboucquez) [reply generators/java/JavaHelper.py]: Added 'not in' support to java and typescript

File: test/vector/transactions.yml (hunk 3 above)
Comment (fboucquez) [reply test/vector/transactions.yml:2]: The original vector tests used private networks. The new schemas removed them so the vector tests needed to be recreated. This is the same file you have in the dev branch gimre has updated (I did the same in another experimental branch).

https://github.com/symbol/catbuffer-generators/blob/dev/test/vector/transactions.yml

**Comment by fboucquez** [comment]
Hi @gimre-xymcity , have you merged the dev branch? It invalidates my hot fix for the revokable mosaics. I can publish a hot fix from this branch before merging though...

**Comment by fboucquez** [comment]
Hi @Jaguar0625 @gimre-xymcity 

Can I move out these 3 commits from the main branch into an experimental? 

image

https://github.com/symbol/catbuffer-generators/commits/main

They were going to be used in the new TS SDK before the schema updates, YML export/import, and the decision to move the generators out of this project into each SDK.

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**